frontend/src/actionCreators/staticContentActionCreator.js

```diff
@@ -114,7 +114,7 @@ export const fetchMineIncidentFollowActionOptions = () => (dispatch) => {
       dispatch(success(reducerTypes.GET_MINE_INCIDENT_FOLLOWUP_ACTION_OPTIONS));
       dispatch(staticContentActions.storeMineIncidentFollowActionOptions(response.data));
     })
-    .catch(() => dispatch(error(reducerTypes.GET_MINE_INCIDENT_FOLLOW_ACTION_OPTIONS)));
+    .catch(() => dispatch(error(reducerTypes.GET_MINE_INCIDENT_FOLLOWUP_ACTION_OPTIONS)));
```

Comment:
One thing I noticed is that now - after the updates - the compiler gets a lot more stringent on imports. Found a fair few typos that I fixed as part of this.